bigquery/indexer.py

```diff
@@ -207,7 +203,18 @@ def index_table(es, index_name, table_name, participant_id_col, sample_id_col,
 
     elapsed_time = time.time() - start_time
     elapsed_time_str = time.strftime("%Hh:%Mm:%Ss", time.gmtime(elapsed_time))
-    logger.info('pandas -> ElasticSearch index took %s' % elapsed_time_str)
+    logger.info('dataframe -> ElasticSearch index took %s' % elapsed_time_str)
+
+
+def index_fields(es, index_name, table):
+    field_docs = _field_docs_by_id(table.full_table_id, table.schema)
+    indexer_util.bulk_index_docs(es, index_name, field_docs)
+
+
+def read_table(bq, table_name):
+    project_id, dataset, table = table_name.split('.')
+    t = bq.get_table(bq.dataset(dataset, project=project_id).table(table))
```

Comment:
Normally we [don't abbreviate in Python](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#316-naming)

Here you can just do `return bq.get_table(bq.dataset(dataset, project=project_id).table(table))`